File: backend/api/Models/ServerEvents/ServerNotifiesClientsInRoomSomeoneHasLeftRoom.cs
using api.Models.DbModels;

namespace api.Models.ServerEvents;

public class ServerNotifiesClientsInRoomSomeoneHasLeftRoom : BaseDto
{
    public EndUser user { get; set; }

    public int roomId { get; set; }

    public string? message { get; set; }
}

File: frontend/src/app/models/serverNotifiesClientsInRoomSomeoneHasLeftRoom.ts
import {EndUser} from "./entities";
import {BaseDto} from "./baseDto";


export class ServerNotifiesClientsInRoomSomeoneHasLeftRoom extends BaseDto<ServerNotifiesClientsInRoomSomeoneHasLeftRoom>{
  user?: EndUser
  roomId?: number;
  message?: string;
}
